src/v/iceberg/BUILD

```diff
@@ -914,3 +914,19 @@ redpanda_cc_library(
         "//src/v/json",
     ],
 )
+
+redpanda_cc_library(
+    name = "retry_policy",
+    srcs = [
+        "rest_client/retry_policy.cc",
+    ],
+    hdrs = [
+        "rest_client/retry_policy.h",
+    ],
+    include_prefix = "iceberg",
+    deps = [
+        ":rest_client_types",
+        "//src/v/http",
+        "//src/v/net",
+    ],
+)
```

Comment:
nit: same as last commit, let's move these into the `rest_client` directory.

Reply:
done